app.js

```diff
@@ -34,6 +34,10 @@ fs.readdir('./src/events/', (err, files) => {
     return true;
 });
 
+// load the httpServer
+httpServer.setPort(config.http.port);
```

Comment:
Might be worth actually passing the port into the module when it's created, not 100% sure on that though.

Reply:
As discussed we will be keeping the setter for this. If anyone has a better idea of how to manage httpServer config please share.